explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -351,18 +323,20 @@ object SpectroscopyModesTable {
         }
       )
       // itc results cache
-      .useState(
-        ItcResultsCache(Map.empty[ItcResultsCache.CacheKey, EitherNec[ItcQueryProblems, ItcResult]])
+      .useSerialStateView(
```

Comment:
Is it serial, because the counter is now handled in the View, right?

Reply:
Yes